### PR TITLE
[skyr-url] Add new port

### PR DESCRIPTION
**Pull request by glynos**
This port is for Skyr URL, and corresponds to issue #10462

### CODE REVIEW

File: ports/skyr-url/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Moves all .cmake files from /debug/share/skyr-url/ to /share/skyr-url/
+## See /docs/maintainers/vcpkg_fixup_cmake_targets.md for more details
+#vcpkg_fixup_cmake_targets(CONFIG_PATH cmake TARGET_PATH share/skyr-url)
+
+# Handle copyright
+file(INSTALL ${SOURCE_PATH}/LICENSE_1_0.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/skyr-url RENAME copyright)
+
+# Post-build test for cmake libraries
+vcpkg_test_cmake(PACKAGE_NAME skyr-url)
```
Comment (LilyWangL) [inline ports/skyr-url/portfile.cmake]: Thanks for your PR. `vcpkg_test_cmake` is deprecated. Please remove it.

**Review by LilyWangL — CHANGES_REQUESTED** [review]

File: ports/skyr-url/portfile.cmake (hunk 1 above)
Comment (glynos) [reply ports/skyr-url/portfile.cmake]: Thanks for the review. I removed this line.

**Comment by LilyWangL** [comment]
@glynos Thanks for your PR. This port build failed on CI test. You can click the **Detail** to get the error log on CI test. Could you continue to fix this PR?

**Comment by glynos** [comment]
Hello,

The only remaining error I need to fix is described in this log:

[/ci/myagent/_work/1/s/buildtrees/skyr-url/install-x64-linux-dbg-out.log
](https://dev.azure.com/vcpkg/public/_build/results?buildId=32261&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=617275de-25b5-5eb9-57d8-e307df514032&l=1354)

in the "test modified ports" step. Could you tell me how I can access that?

Many thanks for your help.

**Comment by LilyWangL** [comment]
You can click **1 artifact** to get the error log as below:
image

**Comment by glynos** [comment]
Thanks for all your help. I've updated my own code, but now the CI failures all seem to be coming from another library that I don't control:

https://dev.azure.com/vcpkg/public/_build/results?buildId=32447&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=a4822bc1-f3a9-5984-c941-6c1b7aa2ab81&l=18

**Comment by LilyWangL** [comment]
> Thanks for all your help. I've updated my own code, but now the CI failures all seem to be coming from another library that I don't control:
> 
> https://dev.azure.com/vcpkg/public/_build/results?buildId=32447&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=a4822bc1-f3a9-5984-c941-6c1b7aa2ab81&l=18

It is not a related error with this PR. This is CI baseline issue, the related PR #10603.

**Comment by glynos** [comment]
Thanks for the explanation. What do I need to do for this PR? Wait until that one is fixed and try again?

**Review by LilyWangL — APPROVED** [review]

**Comment by dan-shaw** [comment]
Thanks for the PR!

**Pull request merged**